attestation-manager/src/pkg/api/cordon_drain_node.go

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2025 Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+package api
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	// v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+
+	// "k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	// "k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/rest"
+	// "k8s.io/client-go/util/retry"
+	"github.com/open-edge-platform/trusted-compute/attestation-manager/src/pkg/logging"
+)
+
+func CordonAndDrainNode() bool {
+
+	logging.Debug("Cordon and drain node started")
+	nodeName := os.Getenv("NODE_NAME")
+	logging.Info("Cordoning ,Node name: ", nodeName)
+	// Load kubeconfig
+	// kubeconfig := filepath.Join(os.Getenv("HOME"), ".kube", "config")
+	// config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	// if err != nil {
+	// 	log.Fatalf("Error building kubeconfig: %s", err.Error())
+	// }
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		logging.Error("Error building in-cluster config: %s", err.Error())
+	}
+
+	// Create clientset
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		logging.Error("Error creating Kubernetes client: %s", err.Error())
+	}
+
+	// Cordon the node
+	if err := cordonNode(clientset, nodeName); err != nil {
+		logging.Error("Error cordoning node: %s", err.Error())
+		return false
+	}
+	logging.Debug("Node cordoned successfully")
+	// Sleep for a while before the next attestation check
+	logging.Debug("Sleeping for 1 minutes and calling drainNode()")
```

Comment:
why is there a need to sleep for 1 minute before draining the node?  seems arbitrary